### ISSUE TITLE
Content with refreshControl and padder wrong paddings

### ISSUE THREAD

**Issue by ppozniak** [GeekyAnts/NativeBase#1720]
When using `Content` component with `padder` attribute everything is working as excepted. However adding `refreshControl` to this makes the left padding disappear.

Screenshot.
![image](https://user-images.githubusercontent.com/13921243/37824253-4e59aca2-2e8c-11e8-8b1d-06581cf5eeba.png)

## react-native, react and native-base version
0.54.2, 16.3.0-alpha.1, 2.3.7

**Comment by akhil-geekyants** [comment]
@ppozniak checked this. It is working fine for me . Attaching a gif.

Gif

![refreshcontrol](https://user-images.githubusercontent.com/27768184/37890714-3949929a-30ef-11e8-8148-44514b0397c8.gif)

**Comment by ppozniak** [comment]
@akhil-geekyants Made a new snack: https://snack.expo.io/Hy91JEL9f
It seems to **work okay on iOS** however not on android:
![image](https://user-images.githubusercontent.com/13921243/37895091-9a884e16-30e0-11e8-9e76-87321743a3d9.png)

**Comment by akhil-geekyants** [comment]
@ppozniak same issue happens with react-native ScrollView. 
But you can fix this by removing the padder and setting the padding to `contentContainerStyle` of `<Content/>`
```
<Content contentContainerStyle={{ padding: 10 }} refreshControl={<RefreshControl/>}>
```
We will be fixing this. Let us know if you are facing any issue.

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.4.2](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.4.2)
